conf/repos.yaml

```diff
@@ -0,0 +1,11 @@
+REPOS:
```

Comment:
@jyejare There is no harm in keeping these files, In the end,  All these files would be overridden by the  `openshift/conf` and I do not want to create confusion by putting un-necessary repos detail here. As you know, we use only these mentioned repositories in the upgrade and it's better to keep it as it is.

Why we keep a single file for all? Even if, I put it all in a single file and point that file only in the dynaconf then how would I get other conf details because in the  `openshift/conf`  we have different files based on their functionality, and in the upgrade, we are pointing only a single file.